htdocs/core/class/extrafields.class.php

```diff
@@ -956,7 +956,8 @@ private function update_label($attrname, $label, $type, $size, $elementtype, $un
 	 * 	@param	string		$elementtype		Type of element ('all' = all or $object->table_element like 'adherent', 'commande', 'thirdparty', 'facture', 'propal', 'product', ...).
 	 * 	@param	boolean		$forceload			Force load of extra fields whatever is status of cache.
 	 *  @param  string		$attrname           The name of the attribute.
-	 *  @return array{}|array{label:array<string,string>,type:array<string,string>,size:array<string,string>,default:array<string,string>,computed:array<string,string>,unique:array<string,int>,required:array<string,int>,param:array<string,mixed>,perms:array<string,mixed[]>,list:array<string,int>|array<string,string>,pos:array<string,int>,totalizable:array<string,int>,help:array<string,string>,printable:array<string,int>,enabled:array<string,int>,langfile:array<string,string>,css:array<string,string>,csslist:array<string,string>,hidden:array<string,int>,mandatoryfieldsofotherentities?:array<string,string>,loaded?:int,count:int}		Array of attributes keys+label for all extra fields.  Note: count set as present to avoid static analysis notices
+	 *  @return array<string>					Array of attributes keys+label for all extra fields.  Note: count set as present to avoid static analysis notices
```

Comment:
IMHO the new return value is array<string,string> .
If not specifed as such, there will be notices indicating that an array key to access the result is a string while an int is expected.

Reply:
good point, I corrected!